index.bs

```diff
@@ -222,8 +222,9 @@ The family of {{Element/setHTML()}}-like methods all accept an options
 dictionary. Right now, only one member of this dictionary is defined:
 
 <pre class=idl>
+enum SanitizerPresets { "default" };
 dictionary SetHTMLOptions {
-  (Sanitizer or SanitizerConfig) sanitizer = {};
+  (Sanitizer or SanitizerConfig or SanitizerPresets) sanitizer = "default";
```

Comment:
This does not work as it would make the behavior the same for `setHTMLUnsafe()` as `setHTML()`. I think we only want to give the latter SanitizerPresets for now and the default.

This also makes me think of a slightly different API shape. What if the `Sanitizer` constructor took `SanitizerPresets`? Then we would not need `Sanitizer.getDefault()` and possibly other static methods in the future. The only problem is that "`default`" would no longer be a good name as the default `Sanitizer` is `{}`.

Reply:
> This does not work as it would make the behavior the same for `setHTMLUnsafe()` as `setHTML()`. I think we only want to give the latter SanitizerPresets for now and the default.

Oh, I overlooked that. Is there a way to do that without copying SetHTMLOptions, or will I need two of those? (`SetHTMLOptions`/`SetHTMLUnsafeOptions`).
 
> This also makes me think of a slightly different API shape. What if the `Sanitizer` constructor took `SanitizerPresets`? Then we would not need `Sanitizer.getDefault()` and possibly other static methods in the future. The only problem is that "`default`" would no longer be a good name as the default `Sanitizer` is `{}`.

I guess `(SanitizerConfig or SanitizerPresets)`? I think that makes sense.

Reply:
Uploaded a new draft, with SetHTMLOptions and SetHTMLUnsafeOptions. Also changed the constructor to take a config or presets.